### ISSUE TITLE
Feature Request : Display user name against Assistant

### ISSUE THREAD

**Issue by kulbinderdio** [bionic-gpt/bionic-gpt#601]
On Assistants screen display a username against each Assistant, and then have ability to show all Assistants by that user, e.g. by clicking on their username

**Comment by kulbinderdio** [comment]
Ability to search for Assistants??

**Comment by tbendien** [comment]
Yes, we need to work on getting sort, filter and search enabled in the Assistants page. There are so many Assistants being created that people are getting overwhelmed and find it hard to find what they need. Suggest we add a ability to add meta tags to Assistants which can be used to filter/sort Assistants. Also, being and to filter based on Assistant owner as mentioned above.
